src/webhooks/options-automator/options-automator.ts

```diff
@@ -26,15 +26,49 @@ export async function sendOptionAutomatorUpdatesToDataDog(
   message: OptionsAutomatorResponse,
   timestamp: number
 ) {
-  // Not sure how detailed we want the text message to be, but this should be fine initially
-  const params: v1.EventCreateRequest = {
-    title: 'Options Automator Update',
-    text: JSON.stringify(message),
-    alertType: 'info',
-    dateHappened: timestamp,
-    tags: [`sentry_region:${message.region}`],
+  const formatRegionTag = (region: string): string => {
+    if (region === 'us' || region === 'de') {
+      return `sentry_region:${region}`;
```

Comment:
would be nice to have a test to ensure that `us` or `de` will be handled properly